lib/src/rules/unnecessary_library_directive.dart

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2015, the Dart project authors. Please see the AUTHORS file
```

Comment:
=> 2022

Reply:
You can use the suggest change feature, too – very nice!